### PR TITLE
Fixes #1676 PI Export for R (dev only)

### LINKED ISSUE #1676: PI Export for R (dev mode only)

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Review by msevestre — COMMENTED** [review]
@Yuri05
Quick implementation to get it going. 
I am exporting PI and Obs Data as json and pkml in sub folders
Export Simulation as pkml

File: src/PKSim.Presentation/Presenters/ContextMenus/ParameterIdentificationContextMenu.cs [hunk 1]
```diff
@@ -16,7 +20,19 @@ public ParameterIdentificationContextMenu(ParameterIdentification parameterIdent
...
-         return ParameterIdentificationContextMenuItems.ContextMenuItemsFor(parameterIdentification);
+         return new List<IMenuBarItem>(ParameterIdentificationContextMenuItems.ContextMenuItemsFor(parameterIdentification))
+         {
+            exportParameterIdentificationToR(parameterIdentification)
+         };
+      }
+
+      private IMenuBarItem exportParameterIdentificationToR(ParameterIdentification parameterIdentification)
+      {
+         return CreateMenuButton.WithCaption(PKSimConstants.MenuNames.AsDeveloperOnly("Export to R®"))
```
Comment (Yuri05) [inline src/PKSim.Presentation/Presenters/ContextMenus/ParameterIdentificationContextMenu.cs:31]: "Export **for** R" is probably better

File: src/PKSim.Presentation/UICommands/ExportParameterIdentificationToRUICommand.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+         if (string.IsNullOrEmpty(folder))
+            return string.Empty;
+
+         var newDirectoryName = Subject.Name;
+         folder = Path.Combine(folder, newDirectoryName);
+
+         if (!DirectoryHelper.DirectoryExists(folder))
+            return DirectoryHelper.CreateDirectory(folder);
+
+         if (_dialogCreator.MessageBoxYesNo(Captions.DoYouWantToDeleteDirectory(newDirectoryName), Captions.Delete, Captions.Cancel) == ViewResult.No)
```
Comment (Yuri05) [inline src/PKSim.Presentation/UICommands/ExportParameterIdentificationToRUICommand.cs:47]: Default button for such a "dangerous" action should be "No"

Comment (msevestre) [reply src/PKSim.Presentation/UICommands/ExportParameterIdentificationToRUICommand.cs:47]: sure. Copy paste from the code in ospsuite.core for Matlab export. I guess no one cared too much before :)

File: src/PKSim.Presentation/Presenters/ContextMenus/ParameterIdentificationContextMenu.cs (hunk 1 above)
Comment (msevestre) [reply src/PKSim.Presentation/Presenters/ContextMenus/ParameterIdentificationContextMenu.cs]: done

**Pull request merged**